dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3241,98 +3387,304 @@ def snapshot_page(n_clicks, url, wait_selector):
+
+        fatal: Could not read from remote repository.
+
+        Please make sure you have the correct access rights
+
+        and the repository exists.
+        ```
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        If deploying with SSH, then this can happen when your `~/.ssh/config` is improperly configured. See  <dccLink href="/dash-enterprise/ssh" children="ssh"/> for more details.
+        
+        If deploying with HTTPS, then this can happen if your username or password is incorrect. Note that usernames and passwords in Dash Enterprise are case sensitive; e.g. the username `Chris.Parmer` is different than the username `chris.parmer`.
+        Git can sometimes cache these usernames after you've typed them. Run `$ git config credential.helper` to verify and correct the username. 
+        Also, some operating system's password managers can cache usernames as well. Open your operating system's password manager and verify the username & password if you are deploying with `https`, running into access denied errors, and `git` isn't asking you for your username and password.
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ### fatal: invalid server response; got 'ed20603d685403ed53ba0017dd8dda3f11407ce6 HEAD'
+
+        ```
+        $ git clone https://<dash-enterprise-server/GIT/<dash-app-name>
+        Cloning into <dash-app-name>...
+        Username for 'https://dash.local':
+        Password for 'https://admin@dash.local':
+
+        fatal: invalid server response; got 'ed20603d685403ed53ba0017dd8dda3f11407ce6 HEAD'
+        ```
+
+        This error can happen if you are using a version of `git` that
+        is incompatible with Dash Enterprise's `git` server.
+        In `git` `v2.26`, `git` switched the default transportation protocol
+        to Version 2. Dash Enterprise's git server runs on `git` 2.17.1
+        and it only supports version `0`. According to the git documentation,
+        the server and client are supposed to be able to negotiate the
+        protocol but, for some as of yet unknown reason, this does not work
+        when deploying with `https`.
+
+        To fix, manually configure `git` to use the old protocol,
+        either in the operation itself:
+        ```
+        $ git -c protocol.version=0 git clone https://<dash-enterprise-server/GIT/<dash-app-name>
+        ```
+        or globally:
+        ```
+        git config --global protocol.version 0
+        ```
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ### Proc entrypoint worker does not exist. Please check your Procfile
+
+        ```
+        $ git push plotly master
+        [...]
+        =======> <dash-app> worker container output:
+        Proc entrypoint worker does not exist. Please check your Procfile
+        ```
+
+        This can happen if the directives listed in `DOKKU_SCALE` don't match
+        the directives listed in `Procfile`.
+
+        For example:
```

Comment:
```suggestion
        For example:
        
        
```

![image](https://user-images.githubusercontent.com/1280389/100397078-f652cc80-3015-11eb-9fbc-2a8e5d773453.png)